.docker/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM composer/composer:latest-bin AS composer
+
+FROM ghcr.io/shyim/wolfi-php/frankenphp:8.3 AS base
+ARG UID=10001
+ARG GID=10001
+RUN <<SH
```

Comment:
Does this multiple command notation cache well?
Never used such approach

Reply:
It creates a single layer instead of multiple layers. It is more efficient but might take longer if previous layers are pretty much static.